ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Features
+   * Add support for alternative implementation for CCM. Add MBEDTLS_CCM_ALT
+     to config.h
+
```

Comment:
Could you add a half-sentence saying that `MBEDTLS_CCM_ALT` controls the usage of alternative CCM implementations? Something like "Add support for alternative implementation for CCM, controlled by new configuration flag `MBEDTLS_CCM_ALT` in `config.h`".